src/components/DataView/DataViewTitleBar/DataViewTitleBarTypes.ts

```diff
@@ -1,13 +1,12 @@
-import { MosaicObject } from "@root/types";
-import { DataViewProps, DataViewSort } from "../DataViewTypes";
+import { DataViewProps } from "../DataViewTypes";
 
 interface SavedViewState {
-	limit: number;
-	sort: DataViewSort;
-	display: string;
-	filter: MosaicObject;
-	activeFilters: string[];
-	activeColumns: string[];
+	limit?: DataViewProps["limit"];
```

Comment:
Looking at the code and the commits, it seems this interface was recently created and there was another one previously created in DataViewTypes.ts so in order to not have duplicates I deleted this one and imported the other directly here.